### PR TITLE
start-stop vertica tasks, and fix to vertica recovery

### PR DESCRIPTION
**Pull request by oneilcin**
This modifies the start and stop tasks for vertica to use
the admintools which is a more reliable way to start and 
stop the db cluster.

I fixed a few problems that were found when using vertical in 
minimon:
The vertica_recovery.j2 template was using the wrong name for
the dbadmin password.
I made the recovery script location configurable so that
it will work in mini-mon.  The default is /opt/monasca, but
this needs to be set to '/vagrant' in mini-mon.

### CODE REVIEW

**Comment by tkuhlman** [comment]
Looks good

File: tasks/start.yml [hunk 1]
```diff
@@ -1,5 +1,12 @@
...
+  register: status_start
+  failed_when: status_start | failed
+  run_once: true
+
+- name: wait for vertica port
+  wait_for: port={{vertica_client_port}} host='127.0.0.1' state=started timeout={{monasca_wait_for_period}}
+
 - name: Start vertica_agent
-  service: name=vertica_agent state=started
+  service: name=vertica_agent state=started
```
Comment (mhoppal) [inline tasks/start.yml]: new line

File: tasks/stop.yml [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
-  service: name=verticad state=stopped
+- name: Stop vertica DB cluster (admintools stop_db)
+  command: su dbadmin -c '/opt/vertica/bin/admintools -t stop_db -d {{ vertica_database_name }} -p "{{vertica_dbadmin_password}}"'
+  register: status_stop
+  failed_when: status_stop | failed
+  run_once: true
+
 - name: Stop vertica_agent
-  service: name=vertica_agent state=stopped
+  service: name=vertica_agent state=stopped
```
Comment (mhoppal) [inline tasks/stop.yml]: new line

**Comment by mhoppal** [comment]
Besides the new lines the changes look good, thank you for fixing those errors on the template I pushed up.

**Pull request merged**